### DISCUSSION TITLE
new interaction

### DISCUSSION THREAD

**Discussion by heartmacaron** [Brionjv/Tomodachi-Life-Save-Editor#95] [General]
hi, i've been using this save editor ever since i got tomodachi life a few months ago and its been working great so far :)!! sorry if i'm doing anything wrong or putting this in the wrong place, i kinda just made a github account now for this topic >.< but i'd like to tell you that i've extracted an interaction not yet in the editor!! it's the one for a mii wanting to get back together with their ex-spouse. would you like me to give you the interaction's file so hopefully you could make use of it? i can put it here or email it if you wish!! though if you wouldn't want me to share it with you for any reason, it's alright ^_^ i wish you good luck with your programs!!

**Comment by Brionjv** [comment]
Hi, yes you can share it, if it's work he will be added
